src/fire.py

```diff
@@ -0,0 +1,43 @@
+import vtk
+
+def create_layer(port, isovalue, r, g, b, alpha):
+    contours = vtk.vtkContourFilter()
+    contours.SetInputConnection(port);
+    contours.SetValue(0, isovalue)
+
+    lut = vtk.vtkLookupTable()
+    lut.SetNumberOfTableValues(1)
+    lut.SetTableValue(0, r, g, b, alpha)
+    lut.Build()
+    
+    mapper = vtk.vtkDataSetMapper()
+    mapper.SetLookupTable(lut)
+    mapper.SetInputConnection(contours.GetOutputPort())
+    
+    actor = vtk.vtkActor()
+    actor.SetMapper(mapper)
+    actor.GetProperty().SetOpacity(alpha)
+    
+    return actor
+
+def add_flame_actor(port, renderer):
+    isosurfaces = [
+                   [400,0.2,0,0,0.7],
+                   [500,0.4,0,0,0.7],
+                   [600,0.6,0,0,0.7],
+                   [700,0.8,0,0,0.7],
+                   [800,1.0,0,0,0.7],
+                   ]
+                   
+    for param in isosurfaces:
+        actor = create_layer(port, *param)
+        renderer.AddActor(actor)
+        
+def add_smoke_actor(port, renderer):
+    isosurfaces = [
+                   [350,1,1,1,0.5],

```

Comment:
I suggest we can use a lower isovalue to capture a more significant smoke range visually.

```suggestion
                   [310,1,1,1,0.5],
```

![image](https://user-images.githubusercontent.com/14951000/232313433-fb65deb9-9eb4-4de9-b339-073a9cd63067.png)

It's just a suggestion. Feel free to ignore this one.